kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -192,7 +202,8 @@ public BatchWriterManager(BigQueryWriter bigQueryWriter,
     private void addNewBatchWriter(TableId tableId) {
       BatchWriter<RowToInsert> batchWriter;
       try {
-        batchWriter = batchWriterConstructor.newInstance(bigQueryWriter);
+        batchWriter = batchWriterConstructor.newInstance();
```

Comment:
I don't think you need batchWriterConstructor at all. You should be able to do `batchWriterClass.newInstance()` directly, and eliminate the whole getBatchWriterConstructor method.